<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1396</width>
    <height>719</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>120</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>tuna</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>280</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>230</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>for_tuna</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>120</y>
      <width>271</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Begin Date</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::MonthSection</enum>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="currentSectionIndex">
       <number>0</number>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>End Date</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>41</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Sync</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Fish Type</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>20</y>
        <width>221</width>
        <height>19</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="Allfish_filter_checkbox">
         <property name="text">
          <string>All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Marlin_filter_checkbox">
         <property name="text">
          <string>Marlin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Shark_filter_checkbox">
         <property name="text">
          <string>Shark</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Tuna_filter_checkbox">
         <property name="text">
          <string>Tuna</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Ship Type</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>20</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>shark</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>751</width>
      <height>571</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QQuickWidget" name="quickWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>491</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="resizeMode">
       <enum>QQuickWidget::SizeRootObjectToView</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Map</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>751</width>
        <height>521</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_chart">
       <attribute name="title">
        <string>Month Chart</string>
       </attribute>
       <widget class="QFrame" name="ChartFrame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>721</width>
          <height>481</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="Cha"/>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Type Chart</string>
       </attribute>
       <widget class="QFrame" name="ChartFrame2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>721</width>
          <height>481</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5"/>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Table</string>
       </attribute>
       <widget class="QTableWidget" name="twTable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>731</width>
          <height>451</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ship ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fish Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Length</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Latitude</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Longtitude</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <zorder>tabWidget</zorder>
     <zorder>pushButton_7</zorder>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Perspective</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>340</y>
      <width>271</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Month</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>20</y>
        <width>179</width>
        <height>67</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox01">
         <property name="text">
          <string>01</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox02">
         <property name="text">
          <string>02</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkBox03">
         <property name="text">
          <string>03</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="checkBox04">
         <property name="text">
          <string>04</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="checkBox05">
         <property name="text">
          <string>05</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox06">
         <property name="text">
          <string>06</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="checkBox07">
         <property name="text">
          <string>07</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="checkBox08">
         <property name="text">
          <string>08</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="checkBox09">
         <property name="text">
          <string>09</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkBox10">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="checkBox11">
         <property name="text">
          <string>11</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QCheckBox" name="checkBox12">
         <property name="text">
          <string>12</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>RESET</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1396</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menutest">
    <property name="title">
     <string>test</string>
    </property>
    <addaction name="actionreset"/>
    <addaction name="actionclear"/>
   </widget>
   <addaction name="menutest"/>
  </widget>
  <action name="actionreset">
   <property name="text">
    <string>reset</string>
   </property>
  </action>
  <action name="actionclear">
   <property name="text">
    <string>clear</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
